src/main.zig

```diff
@@ -1065,6 +1066,13 @@ fn buildOutputType(
                         mem.startsWith(u8, arg, "-I"))
                     {
                         try clang_argv.append(arg);
+                    } else if (mem.startsWith(u8, arg, "-mexec-model=")) {
+                        const value = arg["-mexec-model=".len..];
+                        if (std.mem.eql(u8, value, "reactor")) {
+                            wasi_exec_model = .reactor;
+                        } else if (std.mem.eql(u8, value, "command")) {
+                            wasi_exec_model = .command;
+                        }
```

Comment:
Missing an error when a value other than "reactor" or "command" is specified. I suggest using `std.meta.stringToEnum` (look for other usages of that function nearby).